backend/src/main/resources/application-dev.properties

```diff
@@ -36,4 +36,4 @@ spring.mail.port=587
 spring.mail.username=${SPRING_BOOT_EMAIL}
 spring.mail.password=${SPRING_BOOT_EMAIL_PASSWORD}
 spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
+spring.mail.properties.mail.smtp.starttls.enable=true
```

Comment:
We should keep the newline at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795

Reply:
Good point! I'll make the changes.